src/queue-list/laboratory-tabs.component.tsx

```diff
@@ -71,6 +95,11 @@ const LaboratoryQueueTabs: React.FC = () => {
                 <CompletedList fulfillerStatus={"COMPLETED"} />
               </div>
             </TabPanel>
+            <ExtensionSlot name={labPanelSlot}>
```

Comment:
Am also hoping adding the check won't render this in the view unless there's an existing extension selected

Reply:
thanks Pius ....
by default its not rendering this in the view unless the corresponding extension has been been selected,